### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by CSenshi**
This PR contains adding code style/format checking using pep8 and balck(#11)

### CODE REVIEW

**Comment by CSenshi** [comment]
It seems that code that is already written doesn't satisfy standards implemented.Needs to be changed

File: Makefile [hunk 1]
```diff
@@ -10,6 +10,10 @@ ifdef TEST
 	DOCEST_TEST=validator/rules_src/$(TEST).py
 endif
 
+ifdef PEP8
```
Comment (beqakd) [inline Makefile:13]: This features about style is good! Just let them know to pip install pep8 and black in advance.
Code change needed.

Comment (CSenshi) [reply Makefile:13]: I agree, we should inform users how to install packages, but an issue is already open for CONTRIBUTE.md (#7) which will include all of make commands (all of installations are done using makefile) . So this feature needs separate PR.

Comment (beqakd) [reply Makefile:13]: Noted, thanks. Agreed this one is separate one.

**Pull request merged**